pinot-core/src/main/java/com/linkedin/pinot/core/common/DataFetcher.java

```diff
@@ -198,6 +198,27 @@ public void fetchIntValues(String column, int[] inDocIds, int inStartPos, int le
   }
 
   /**
+   * Fetch the int values for a multi-valued column.
+   *
+   * @param column column name.
+   * @param inDocIds dictionary Id array.
+   * @param inStartPos input start position.
+   * @param length input length.
+   * @param outValues value array buffer.
+   * @param outStartPos output start position.
+   */
+  public void fetchIntValues(String column, int[] inDocIds, int inStartPos, int length, int[][] outValues, int outStartPos) {
```

Comment:
Yes, will should address MV cleanup before any further changes, after this PR. Lets discuss offline.